docs/src/domace/ju1ce/PasovneMatrike.md

```diff
@@ -0,0 +1,34 @@
+ 
+## Domača naloga 1: pasovne matrike
+
+### Podatkovni tipi
+
+Podatkovi tip PasovnaMatrika hrani vektor vrednosti na diagonali, vektor vektorjev vrednosti pod diagonalo in vektor vektorjev vrednosti nad diagonalo. ZgornjePasovnaMatrika in SpodnjePasovnaMatrika sta enaka, le da hranita vektor vektorjev vrednosti le nad/pod diagonalo.
+
+PasovnaMatrika([1,1,1],[[2,2],[3]],[4,4])
```

Comment:
Uporabite raje format besedila brez oblikovanja za kodo in rezultate. V Markdownu lahko uporabite tri narekovaje ```